app/api/auth/[...nextauth]/auth.ts

```diff
@@ -22,16 +24,44 @@ export const {
     }),
   ],
   callbacks: {
-    async jwt({ token, user, account }) {
+    jwt({ token, user, account }) {
```

Comment:
is the keyword "async" not necessary anymore?

Reply:
I was trying to enrich the session with the Avatar. The problem. on upload Profile pic we would need to login/logout to renew the session. hence we can yes remove it.

Reply:
cleared